openshift-knative-operator/cmd/operator/kodata/knative-serving/1.11/2-serving-core.yaml

```diff
@@ -6450,265 +6631,6 @@ spec:
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: apps/v1
```

Comment:
@nak3 @skonto do we need to add this to the release notes?

Reply:
Yes, I think so. But defer to @skonto who changed it on upstream.

Reply:
Yes we should as it changes the deployment setup. This could break things. For example might be users who scrape individual pods for observability reason.

Reply:
--> https://issues.redhat.com/browse/SRVKS-1155